src/main/java/seedu/address/model/task/Priority.java

```diff
@@ -8,8 +8,8 @@
  */
 public class Priority {
 
-    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority should only contain numbers";
-    public static final String PRIORITY_VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority can take any values";
```

Comment:
Okay sorry, I just remembered that we are allowing all values for `v0.1`. Will add an issue for this for `v0.2`.